controls/cis_rhel8.yml

```diff
@@ -1047,40 +1047,67 @@ controls:
       - package_tftp_removed
 
   - id: 3.1.1
-    title: Verify if IPv6 is enabled on the system (Manual)
+    title: Ensure IPv6 status is identified (Manual)
     levels:
       - l1_server
       - l1_workstation
     status: manual
 
   - id: 3.1.2
-    title: Ensure SCTP is disabled (Automated)
+    title: Ensure wireless interfaces are disabled (Automated)
     levels:
-      - l2_server
-      - l2_workstation
+      - l1_server
     status: automated
     rules:
-      - kernel_module_sctp_disabled
+      - wireless_disable_interfaces
 
   - id: 3.1.3
-    title: Ensure DCCP is disabled (Automated)
+    title: Ensure wireless interfaces are disabled (Automated)
+    levels:
+      - l1_server
+    status: partial
+    rules:
+      - service_bluetooth_disabled
+      # Missing rule package_bluez_removed
```

Comment:
This new rule would probably go to related_rules.

Reply:
I will leave out for now since it is not created.